src/services/expenses.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+let expenses = [];
+
+function init() {
+  expenses = [];
+}
+
+function getAll(userId, category, from, to) {
+  if (userId) {
+    expenses = [expenses.find(expense => expense.userId === +userId)];
+  }
+
+  if (from && to) {
+    expenses = expenses.filter(
+      expense => expense.spentAt >= from && expense.spentAt <= to,
+    );
+  }
+
+  if (category) {
+    expenses = expenses.filter(expense => expense.category === category);
```

Comment:
You are reassigning general variable. Maybe better to create local variable (`filteredExpenses`) for this function.

Reply:
totally agree, you are GETTING values, not updating them